scala-mode2-prettify.el

```diff
@@ -0,0 +1,54 @@
+;;; scala-mode-prettify.el - Extension of scala-mode for prettifying scala symbols/code
+;;; Copyright (c) 2016 Merlin Göttlinger
+;;; For information on the License, see the LICENSE file
+
+(defconst scala-mode-pretty-bool-alist '(
+					 ("<=" . ?≤)
```

Comment:
is it safe to put UTF-8 in the file like this? Is there anything else we need to do to make sure Emacs interprets it correctly? e.g. maybe some header information about encoding.